srcpkgs/txr/template

```diff
@@ -26,11 +26,6 @@ pre_configure() {
 	fi
 }
 
-post_configure() {
-	vsed -i config.make -e "\#^PLATFORM_CFLAGS#s#:=.*\$#:= ${CFLAGS}#"
-	vsed -i config.make -e "\#^PLATFORM_LDFLAGS#s#:=.*\$#:= ${LDFLAGS}#"
-}
-
```

Comment:
I think this is still relevant... Why did you remove it?

Reply:
I think [upstream makefile](http://www.kylheku.com/cgit/txr/tree/Makefile#n34) respects the flags set by the build style now, I believe I put this long ago when it didn't use to.